### PR TITLE
:new:💄 Syllabus Update for Fall 2022

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Update the course syllabus/outline for Fall 2022. 
- Dates/rooms/times
- Office hours formats
- Add testing to topics
- Update lab details
- General formatting 
- Remove old info

### Why

Semester is about to start 😬 

### Additional Notes

@jdelamer, you should have a looksie to make sure everything is correct on your end.

### CODE REVIEW

File: site/outline/outline.rst [hunk 1]
```diff
@@ -55,19 +60,19 @@ Lecture Section 11
...
 ------------------
 
 **Dr. Delamer**
 
-* Mon 10:15 - 11:05 (C1), GM 001
-* Wed 09:15 - 10:05 (C2), GM 001
-* Fri 08:15 - 09:05 (C3), GM 001
+* Mon 10:15 - 11:05 (C1), SCHW 110
+* Wed 09:15 - 10:05 (C2), SCHW 110
+* Fri 08:15 - 09:05 (C3), SCHW 110
```
Comment (jameshughes89) [inline site/outline/outline.rst:75]: Verify this is your room.

File: site/outline/outline.rst [hunk 2]
```diff
@@ -158,142 +166,187 @@ Student Evaluation (Tentative Dates)
...
+The purpose of the weekly labs is to give students hands-on experience. Labs will have required assigned work and will
+revolve around `Kattis <https://open.kattis.com/>`_.
 
 
-* Labs begin on the 1st week of class. 
-* Lab exercises must be completed for marks; attendance and lab completion is mandatory. Each completed lab is worth 1% of your final grade (to a maximum of 10%). It is your responsibility to ensure your attendance and exercise completion is recorded. 
-* There is a total of 10 labs, however there is potential for additional labs. In the case where there are more than 10 labs, the marks for your best 10 will be recorded. 
-* You may only attend the lab session for which you are registered.
-* Students must bring their student identification to all labs. 
+Labs begin on the 1st week of class for lab sections 11 and 12. Lab sections 13, and 14 begin week 2.
```
Comment (jameshughes89) [inline site/outline/outline.rst:222]: @jdelamer Verify.

File: site/outline/outline.rst [hunk 3]
```diff
@@ -158,142 +166,187 @@ Student Evaluation (Tentative Dates)
...
+Office Hours
+============
+
+Office hours are provided to students to facilitate their success --- students are encouraged to come to office hours.
+
+Students must understand that the professors and TAs are not there to provide them answers to problems, but to assist
+students solving their problems.
+
+As per university policy, masking and proper social distancing is required. Students refusing to follow this policy will
+be asked to leave.
```
Comment (jameshughes89) [inline site/outline/outline.rst:268]: @jdelamer Will you be doing inperson?

**Comment by jdelamer** [comment]
I modified my office hours and pushed them in #363.

**Review by jdelamer — APPROVED** [review]

**Pull request merged**